### ISSUE TITLE
Reading incorrect value of the ‘Hide Notification Badges’  toggle

### ISSUE THREAD

**Issue by dvagala** [aydenp/FocusPlsNoBadges#6]
Steps to reproduce:
1. Create a new focus mode on iOS with ‘Hide Notification Badges’ option disabled. The app is behaving correctly.
2. Enable the ‘Hide Notification Badges’ option on iOS on that focus mode. The app noticed this and correctly turn off badges on mac os. 
3. Disable the ‘Hide Notification Badges’ option on iOS on that focus mode. The app is reading incorrect value, it is not able to turn on the notification badges. Trying to switching between the other focus modes and still not reading the correct value.

iOS 15.6.1
Mac Os 12.4

**Comment by dvagala** [comment]
Okay, I've done some testing:

activeMode.configuration == 2   // ‘Hide Notification Badges’ enabled
activeMode.configuration == 1   // ‘Hide Notification Badges’ disabled

The issue is that in the code we are checking against the 0
```
        var shouldHideApplicationBadges: Bool {
            return _hideApplicationBadges != 0
        }
```

I don't know the context, but the quick fix would be
```
        var shouldHideApplicationBadges: Bool {
            return _hideApplicationBadges == 2
        }
```

**Comment by aydenp** [comment]
Interesting -- I was consistently getting 0 and 2 during testing, but this would still make some sense. Maybe I hadn't tested hard enough.

Could you make a PR for this so you get credit?

**Comment by dvagala** [comment]
@aydenp Sure! Here it is #7

**Comment by aydenp** [comment]
Fixed in #7 ^
